### PR TITLE
fix: env update trigger from #1049

### PR DESCRIPTION
**Pull request by lkstrp**
- This should fix #1049 which was never working
- Also the `envs/environment.fixed.yaml` is empty right now?
- And we could think about renaming that to `pinned`

### CODE REVIEW

File: .github/workflows/update-fixed-env.yaml [hunk 1]
```diff
@@ -1,39 +1,34 @@
...
         environment-file: envs/environment.yaml
-        log-level: debug
-        init-shell: bash
-        cache-environment: true
-        cache-downloads: true
 
     - name: Update environment.fixed.yaml
       run: |
-        mamba env export --file envs/environment.fixed.yaml  --no-builds
+        conda env export --name pypsa-eur > envs/environment.fixed.yaml
```
Comment (fneum) [inline .github/workflows/update-fixed-env.yaml:26]: Wouldn't it be better with `--no-builds`?

File: .github/workflows/update-fixed-env.yaml [hunk 2]
```diff
@@ -1,39 +1,34 @@
-name: Fixed Environment YAML Monitor
+name: Environment Checker
```
Comment (fneum) [inline .github/workflows/update-fixed-env.yaml:1]:
```suggestion
name: Fixed-Version Environment Checker
```

**Review by fneum — CHANGES_REQUESTED** [review]
Fine to merge, perhaps `--no-builds`?

File: .github/workflows/update-fixed-env.yaml (hunk 1 above)
Comment (lkstrp) [reply .github/workflows/update-fixed-env.yaml]: Thanks!

**Pull request merged**